queries/helm/highlights.scm

```diff
@@ -0,0 +1 @@
+; inherits: gotmpl
```

Comment:
I guess we can add helm specific functions here later?

Reply:
You are right, that's a good idea. Do you have some in mind that you want to specifically highlight beyond their gotmpl highlighting?